packages/MarkdownEditor-Tests.package/MarkdownBlockParserTest.class/instance/assert.parsesToBlocks..st

```diff
@@ -0,0 +1,9 @@
+accessing
+assert: aCollectionOfStrings parsesToBlocks: aCollectionOfMarkdownBlockClasses
```

Comment:
This was a idea to have multiple lines without ugly line breaks in the code. But if you think that this method is unneccessary overhead, we'll use the simpler solution.

Reply:
I like it except for the fact that the second argument is the expected collection. Usually in Squeak the first argument is the expectation.

Suggestion: `assert: aCollectionOfMarkdownBlockClasses isParsedStructureOf: aCollectionOfLines`

Reply:
Why don't you use `'text', Character linefeed, 'text in new line'`? The below tests are quite hard to understand this way. They don't contain the concept that there are newlines in between the strings.

Reply:
If we would use a "line collection" throughout our project, it would enable us a more compact and understandable definition.
Yes, it hides that there are newlines, but using `'# Heading', Character cr, 'Paragraph'` loses the focus on the content to the newlines imo. (afaik `Character cr` is more broadly used then `Character linefeed`)

Reply:
Then maybe we should consider renaming the convenience test method to make it a little more clear.